components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.h

```diff
@@ -24,6 +24,20 @@
  Delegate for MDCBottomDrawerContainerViewController.
  */
 @protocol MDCBottomDrawerContainerViewControllerDelegate <NSObject>
+
+/**
+This method is called when the bottom drawer updates its value for scrollViewIsScrolledToBottom when
```

Comment:
I already left this comment but it became outdated--I renamed `scrollViewIsScrolledToBottom` to `scrollViewIsScrolledToEndOfContent` so people wouldn't confuse it with the preexisting (and questionably named) `scrollViewIsDraggedToBottom` property, but failed to change it in some of the documentation, including this line. So `scrollViewIsScrolledToBottom` should be changed to `scrollViewIsScrolledToEndOfContent`.